modules/ROOT/pages/mongodb/mongodb-connector.adoc

```diff
@@ -6,47 +6,32 @@ endif::[]
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-MongoDB Connector v5.3
-
 Anypoint Connector for MongoDB is a closed-source connector that provides a connection between the Mule runtime and third-party software on a MongoDB server.
 
 Release Notes: xref:release-notes::connector/mongodb-connector-release-notes-mule-4.adoc[MongoDB Connector Release Notes] +
 Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-mongodb-connector/[MongoDB Connector]
 
-== About Connectors
-
-Anypoint connectors are Mule runtime engine (Mule) extensions that enable you to connect to APIs and resources on external systems, such as Salesforce, MongoDB, ServiceNow, and Twitter.
-
-== Prerequisites
-
-Before creating an app, you must have access to the MongoDB target resource and Anypoint Platform. You must also understand how to create a Mule app using Design Center or Anypoint Studio.
-
-To use the this connector you need:
-
-* MongoDB instance
-* Set of valid credentials, including the required MongoDB endpoints, pointing to your instance
-
-
-== Audience
-
-* Starting user -- To create your Mule app, see 
-xref:mongodb/mongodb-connector-design-center.adoc[Design Center] or 
-xref:mongodb/mongodb-connector-studio.adoc[Anypoint Studio]. 
-* Power user - Read xref:mongodb/mongodb-connector-xml-maven.adoc[XML and Maven Support]
-and xref:mongodb/mongodb-connector-examples.adoc[Examples].
-
-== Exchange Example
+This documentation set consists of the following version documents:
```

Comment:
When I first read this, I was curious about "version documents." Hmm. Documents by Version?